docs/modules/ROOT/pages/usage.adoc

```diff
@@ -152,7 +157,7 @@ Always read the API documentation for each integrated component.
 
 Some components require some sort of setup upon construction.
 Usually, this would be a job for a constructor; however, components themselves cannot provide constructors.
-Components instead offer ``initializer``s within their `InternalImpl` which enables a contract to create a constructor and invoke the component's `initializer`.
+Components instead offer ``initializer``s within their `InternalImpl` to be called by the contract's constructor.
```

Comment:
```suggestion
Components instead offer ``initializer``s within their `InternalImpl` to call from the contract's constructor.
```
i know i suggested the current version but I think these two changes read better